_data/navigation.yml

```diff
@@ -1,115 +1,10 @@
 # main links
 main:
-  - title: "Virtual Conference"
-    url: https://www.virtual2021.eacl.org/
-    highlight: true
-  - title: "Schedule"
-    url: /schedule/
-  - title: "Program"
-    url: /program/keynotes/
-  - title: "News"
-    url: /news
-  - title: "Registration"
-    url: /registration
-  - title: "Sponsors"
-    url: /sponsors
-  - title: "Code of Conduct"
-    url: /code
-  - title: "Committees"
-    url: /committees/organization
   - title: "Calls"
```

Comment:
Consider changing to "Calls for Papers" because there are no other calls for now, and the navigation bar is otherwise quite bare.

Reply:
Good point, it's addressed now.